src/MICore/LaunchOptions.cs

```diff
@@ -1350,6 +1522,28 @@ private IEnumerable<string> GetSOLibSearchPathCandidates()
             }
         }
 
+        protected void InitializeCommonOptions(Json.LaunchOptions.BaseOptions options)
+        {
+            this.ExePath = options.Program;
+
+            if (this.TargetArchitecture == TargetArchitecture.Unknown && !String.IsNullOrWhiteSpace(options.TargetArchitecture))
+            {
+                this.TargetArchitecture = ConvertTargetArchitectureAttribute(options.TargetArchitecture);
+            }
+
+            this.VisualizerFile = options.VisualizerFile;
+            this.ShowDisplayString = options.ShowDisplayString.GetValueOrDefault(false);
+
+            this.AdditionalSOLibSearchPath = String.IsNullOrEmpty(this.AdditionalSOLibSearchPath) ?
+                options.AdditionalSOLibSearchPath :
+                String.Concat(this.AdditionalSOLibSearchPath, ";", options.AdditionalSOLibSearchPath);
+
+            // This was set in OpenDebugAD7 for !CORECLR
```

Comment:
>This [](start = 15, length = 4)

Might be worth adding a TODO to have a launch option for this. It didn't make sense in VS Code since we don't have any UI in VS Code. But it does make sense in VS.